cernopendata/modules/fixtures/data/records/cms-SW-FIXMEfilename.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "abstract": {
+      "description": "This is a CMSSW module for producing flat tuples containing jet properties from 13 TeV Run2 MC samples. The code is intended to run inside the CMS Open Data environment."
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Kallonen, Kimmo",
+        "orcid": "0000-0001-9769-7163"
+      }
+    ],
+    "collections": [
+      "CMS-Validation-Utilities"
```

Comment:
Hmm, this is not validation code, although I see we used it in the past... http://opendata-dev.cern.ch/record/5104

Reply:
... should be rather put under `CMS-Tools` collection

Reply:
OK for `CMS-Tools` (or leave out)

Reply:
Amended